releases/nightly-build/compose-files/docker-compose-nexus-redis.yml

```diff
@@ -0,0 +1,574 @@
+# /*******************************************************************************
```

Comment:
Please rebase on top of upstream/master.  Will try to note specifically the changes that need to be picked up.